docs/snippets/angular/storybook-preview-with-angular-polyfills.js.mdx

```diff
@@ -1,21 +1,24 @@
 ```js
 // .storybook/preview.js
 
+import { Preview } from '@storybook/addon-docs/angular';
```

Comment:
@shilman feel free to correct me but shouldn't this import be coming directly from the `@storybook/angular` package instead of `@storybook/addon-docs`?